tests/dummy/app/pods/index/index-content/template.md

```diff
@@ -0,0 +1,41 @@
+## Why Cucumber
+
+Cucumber is a famous BDD implementation that gives exciting promises.
+
+
+
+#### Acceptance tests in form of <em>user stories</em> offer a lot of benefits:
```

Comment:
Idk why, but this renders as a link, which points to itself. Which is, weird.

Also noticed that these links are rendered with the default blue of the browser, not the blue defined in the addon-docs theme as all the other "real" links...

Reply:
Maybe 4 # does not exist and it sees the leftover # as a hash url?

Reply:
> Idk why, but this renders as a link, which points to itself. Which is, weird.

Headers are intentionally links anchored to themselves! This lets you copy a link to a specific section of the page, not to the top of the page.

Note that it works with pure HTML, no JS involved.

***

> Also noticed that these links are rendered with the default blue of the browser

This is an upstream issue. I have worked around it by using higher-ranked h* tags which are not affected.